public/index.html

```diff
@@ -5,6 +5,7 @@
   <title>Twin Cities Mutual Aid</title>
   <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
   <link rel="icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet">
```

Comment:
Pulling in Material Icons cause it's much easier than having to create new pngs or svgs anytime we want an icon. Also styling icons is way easier this way